docs/source/releases/1.17.2/1.17.2-enhancement-add-x-y-size-args-to-imagery-annotated.yaml

```diff
@@ -0,0 +1,19 @@
+enhancement:
+  - title: Add x_size and y_size kwargs to imagery_annotated output formatter.
+    description: |
+      Allows specifying x_size and y_size separately from the area definition shape.
+      This is useful if you are producing multiple output formats for a single sector,
+      and you do not need as high resolution output for the annotated image as other
+      potential outputs.  This prevents reprocessing the data for multiple sized sectors,
+      but still allows outputting a lower resolution / smaller annotated image.
+
+      This reduced the time to write an 8250x8000 sector from 42s for the full size
```

Comment:
This can probably be further improved,  but allowing writing out a different size is the first step.  It occurred to me after the fact that this also resolves the issue of not being able to plot a larger annotated image for very small sectors (titles don't look great if you are trying to plot like a 50x50 sector ...).